<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1305</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FLEX</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="result_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>1291</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="file_sections">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>541</width>
      <height>201</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Images">
     <attribute name="title">
      <string>Images</string>
     </attribute>
     <widget class="QLabel" name="text_image_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
     <widget class="QComboBox" name="input_type_image">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PNG</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>JPEG</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>JPG</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ICO</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GIF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>JIFI</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PBM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PGM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PPM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BMP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CUR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SVG</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SVGZ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>XMB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>XPM</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="text_image_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>to</string>
      </property>
     </widget>
     <widget class="QComboBox" name="output_type_image">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PNG</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>JPEG</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>JPG</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ICO</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>JIFI</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PBM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PGM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PPM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BMP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CUR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>XBM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>XPM</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="convert_button_image">
      <property name="geometry">
       <rect>
        <x>369</x>
        <y>10</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="image_progress">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>521</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QPushButton" name="convert_button_image_save">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Save As</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="VideoAudio">
     <attribute name="title">
      <string>Video/Audio</string>
     </attribute>
     <widget class="QLabel" name="text_av_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
     <widget class="QComboBox" name="input_type_av">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MP4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MOV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AVI</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WMV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MKV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>M4V</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MP3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WAV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AIFF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WMA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FLAC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ALAC</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="text_av_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>to</string>
      </property>
     </widget>
     <widget class="QComboBox" name="output_type_av">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MP4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MOV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AVI</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WMV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MKV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>M4V</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MP3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WAV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AIFF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>WMA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>FLAC</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ALAC</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="convert_button_av">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="av_progress">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>521</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QPushButton" name="convert_button_av_save">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Save As</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>Documents</string>
     </attribute>
     <widget class="QLabel" name="text_doc_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>to</string>
      </property>
     </widget>
     <widget class="QComboBox" name="input_type_doc">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DOCX</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CSV</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EPUB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>HTML</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>JSON</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LATEX</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ODT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RTF</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="output_type_doc">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>91</width>
        <height>24</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DOCX</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>EPUB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>HTML</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>JSON</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PPTX</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MD</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>PDF</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>LATEX</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ODT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RTF</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="convert_button_doc">
      <property name="geometry">
       <rect>
        <x>369</x>
        <y>10</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="doc_progress">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>521</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
     <widget class="QPushButton" name="convert_button_doc_save">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>80</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Save As</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1305</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionChange_Save_Folder"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionChange_Save_Folder">
   <property name="text">
    <string>Change Save Folder</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
